tests/testapp/tests/test_controller_methods.py

```diff
@@ -358,3 +354,18 @@ def test_delete_model_in_store_deletes_models_in_app(self):
         self.assertFalse(Facility.objects.filter(id=child2.id).exists())
         self.assertFalse(Facility.objects.filter(id=grandchild1.id).exists())
         self.assertFalse(Facility.objects.filter(id=grandchild2.id).exists())
+
+    def test_models_created_successfully(self):
+        root = FacilityModelFactory()
+        child1 = FacilityModelFactory(parent=root)
+        child2 = FacilityModelFactory(parent=root)
+        self.mc.serialize_into_store()
+        Facility.objects.all().delete()
+        DeletedModels.objects.all().delete()
```

Comment:
Maybe do another serialization in here, otherwise there might be stuff left around in `DeletedModels`?